test/unit/drivers/sqlite.test.js

```diff
@@ -73,7 +73,7 @@ describe('=> SQLite driver', () => {
       id: { type: BIGINT, primaryKey: true, autoIncrement: true },
       title: { type: STRING, allowNull: false },
     });
-    await driver.query(`INSERT INTO notes (id, title) VALUES (42, 'Untitled')`);
+    await driver.query('INSERT INTO notes (id, title) VALUES (42, \'Untitled\')');
```

Comment:
These changes are a bit unnecessary. Can the corresponding eslint rule be adjusted?